diff -ur -urp exact-image-0.8.5.orig/api/lua/Makefile exact-image-0.8.5/api/lua/Makefile
--- exact-image-0.8.5.orig/api/lua/Makefile	2008-10-18 11:11:43.000000000 +0200
+++ exact-image-0.8.5/api/lua/Makefile	2011-05-14 00:01:05.000000000 +0200
@@ -16,7 +16,7 @@ X_BUILD_IMPLICIT := 0
 include build/bottom.make
 X_BUILD_IMPLICIT := $(_X_BUILD_IMPLICIT)
 
-api/lua/libdir := $(shell pkg-config lua --variable INSTALL_CMOD)
+api/lua/libdir := $(shell pkg-config lua51 --variable INSTALL_CMOD)
 
 # install
 all:: $($(X_MODULE)_BINARY)
diff -ur -urp exact-image-0.8.5.orig/configure exact-image-0.8.5/configure
--- exact-image-0.8.5.orig/configure	2010-11-21 11:01:04.000000000 +0100
+++ exact-image-0.8.5/configure	2011-05-14 00:22:45.000000000 +0200
@@ -91,7 +91,7 @@ pkgcheck swig shell SWIG 'swig -version 
 
 # supported swig target languages so far
 
-pkgcheck lua pkg-config LUA atleast 5.1
+pkgcheck lua51 pkg-config LUA atleast 5.1
 
 pkgcheck perl shell PERL 'perl -version 2>/dev/null | sed -n "s/This is perl.*v\([0-9.]*\).*built.*/\1/p"' atleast 5.8.0 &&
 	PERLINCS="`perl -MExtUtils::Embed -e ccopts`"
